### DISCUSSION TITLE
Show multiple tooltips onslected for line points.

### DISCUSSION THREAD

**Discussion by vaibhav135** [naver/billboard.js#3053] [Q&A]
Hi,
I want to show multiple tooltips onselected for a point and want it to be visible until the point is unselected. Is there any way to do this. 
Thank you.

**Comment by netil** [comment]
Can you illustrate this in more detail? I can't get exact behaves that you want.

**Reply by vaibhav135** [reply]
@netil sorry I don't have anything to illustrate at this moment. But it's something like this 
- You click a point tooltip will appear 
- then you click on another point a second tooltip will appear on the point clicked (the first tooltip won't disappear). 
- When you click on the third points, both the tooltips will disappear.

**Reply by watnab** [reply]
> When you click on the third points, both the tooltips will disappear.

I could not see why the third points will make other points unselected if "tooltips disappear" means "points are unselected".

----

There can be many `.bb-selected-circle`.
However it seems a chart can have only one `.bb-tooltip-container`.
It might be the element can be cloned manually though.
https://naver.github.io/billboard.js/demo/#Data.DataSelection
